src/main/java/com/couchbase/lite/auth/Utils.java

```diff
@@ -0,0 +1,58 @@
+package com.couchbase.lite.auth;
```

Comment:
Would it be better to have all Utils classes at the same place at `com/couchbase/lite/util/` package?